src/test/java/de/uhd/ifi/se/decision/management/jira/view/TestRequirementsDashboardItem.java

```diff
@@ -1,62 +1,82 @@
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
 import java.util.HashMap;
 import java.util.Map;
 
-import org.junit.Before;
-import org.junit.Test;
+import javax.servlet.http.HttpServletRequest;
 
-import com.atlassian.jira.component.ComponentAccessor;
-import com.atlassian.jira.issue.issuetype.IssueType;
-import com.atlassian.jira.issue.issuetype.MockIssueType;
+import com.atlassian.jira.bc.issue.search.SearchService;
+import com.atlassian.jira.config.ConstantsManager;
+import com.atlassian.jira.config.IssueTypeManager;
+import com.atlassian.jira.issue.IssueManager;
+import com.atlassian.jira.issue.comments.CommentManager;
+import com.atlassian.jira.issue.fields.config.manager.IssueTypeSchemeManager;
+import com.atlassian.jira.issue.link.IssueLinkManager;
+import com.atlassian.jira.mock.MockConstantsManager;
 import com.atlassian.jira.mock.MockProjectManager;
-import com.atlassian.jira.project.MockProject;
-import com.atlassian.jira.project.Project;
-
+import com.atlassian.jira.mock.component.MockComponentWorker;
+import com.atlassian.jira.mock.security.MockAuthenticationContext;
+import com.atlassian.jira.project.ProjectManager;
+import com.atlassian.jira.security.JiraAuthenticationContext;
 import de.uhd.ifi.se.decision.management.jira.TestSetUp;
-import de.uhd.ifi.se.decision.management.jira.config.PluginInitializer;
+import de.uhd.ifi.se.decision.management.jira.extraction.gitclient.TestSetUpGit;
+import de.uhd.ifi.se.decision.management.jira.mocks.MockCommentManager;
+import de.uhd.ifi.se.decision.management.jira.mocks.MockIssueLinkManager;
+import de.uhd.ifi.se.decision.management.jira.mocks.MockIssueManager;
+import de.uhd.ifi.se.decision.management.jira.mocks.MockIssueTypeManager;
+import de.uhd.ifi.se.decision.management.jira.mocks.MockIssueTypeSchemeManager;
+import de.uhd.ifi.se.decision.management.jira.mocks.MockSearchService;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeStatus;
 import de.uhd.ifi.se.decision.management.jira.model.KnowledgeType;
 import de.uhd.ifi.se.decision.management.jira.model.text.PartOfJiraIssueText;
 import de.uhd.ifi.se.decision.management.jira.persistence.singlelocations.JiraIssueTextPersistenceManager;
 import de.uhd.ifi.se.decision.management.jira.testdata.JiraIssues;
 import de.uhd.ifi.se.decision.management.jira.testdata.JiraUsers;
 import net.java.ao.test.jdbc.NonTransactional;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
 
-public class TestRequirementsDashboardItem {
+import static de.uhd.ifi.se.decision.management.jira.testdata.JiraIssues.addElementToDataBase;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class TestRequirementsDashboardItem extends TestSetUp {
 
 	private RequirementsDashboardItem dashboardItem;
 	private Map<String, Object> params;
+	@Mock

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5632096)